zebra-state/src/service/finalized_state/disk_format/tests/snapshot.rs

```diff
@@ -143,12 +145,6 @@ fn snapshot_raw_rocksdb_column_family_data(db: &DiskDb, original_cf_names: &[Str
             // because those roots are used to populate the anchor column families.
             insta::assert_ron_snapshot!(format!("{}_raw_data", cf_name), cf_data);
         }
-
-        assert_eq!(
-            cf_iter.status(),
```

Comment:
`status()` is not available anymore so i just removed this check.

Reply:
It has moved to the DBRawIteratorWithThreadMode type.